protostar/migrator/cheatcodes/migrator_invoke_cheatcode.py

```diff
@@ -25,14 +25,12 @@
 
 class SignedCheatcodeConfig(CheatcodeNetworkConfig):
     max_fee: NotRequired[Wei]
```

Comment:
```suggestion
    max_fee: NotRequired[Fee]
```

Reply:
[change type of max_fee to Fee](https://github.com/software-mansion/protostar/pull/979/commits/8b3d0ca5d7f7c558635efaccc2bc6c78e0f11c82)